content/introduction/eks/eks_high_architecture.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Architecture"
+date: 2018-10-03T10:23:24-07:00
+draft: false
+weight: 150
+---
+
+
```

Comment:
Also, what does this Architecture mean? One line text that specifies, Once you have your EKS cluster ready, this architecture shows communication between control plane and worker nodes

Reply:
I added a sentence describing the diagram.